src/scikit_build_core/_shutil.py

```diff
@@ -44,7 +44,8 @@ def _run(
 
         if self.env:
             logger.debug(
-                "RUNENV:\n  {}", "\n  ".join(f"{k}={self.env[k]}" for k in sorted(self.env.keys()))
+                "RUNENV:\n  {}",
+                "\n  ".join(f"{k}={self.env[k]}" for k in sorted(self.env.keys())),
```

Comment:
```suggestion
                "\n  ".join(f"{k}={v}" for k, v in sorted(self.env.items())),
```

(Minor touchup while this is being updated)